types/requests.go

```diff
@@ -7,3 +7,16 @@ type ScaleServiceRequest struct {
 	ServiceName string `json:"serviceName"`
 	Replicas    uint64 `json:"replicas"`
 }
+
+// IdentityRequest provides information about the underlying provider to the OpenFaaS gateway
```

Comment:
Remove the suffix ` to the OpenFaaS gateway` - this code should be agnostic of what is upstream.